Improve description of coordinate system

ubruhin opened the pull request:
This very important thing was not really good explained until now...

dbrgn commented on SPECIFICATION.md:
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
-point to the right. Positive Y coordinates point to the top. Negative
-coordinates are not allowed.
+point to the right. Positive Y coordinates point to the top. A value of 9 means
+100% of the font height (i.e. the coordinate system is scaled with factor 9).
```
"Font height" is ambiguous. Maybe use the term "cap height"? See https://github.com/fontobene/fontobene/issues/5

We should probably also include that illustration in the spec. We have permission to do so.

dbrgn commented on SPECIFICATION.md:
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
-      (0,0)   (X,0)
+      (0,0)   (9,0)
+
+Typical glyphs use the X axis (Y=0) as the baseline and are drawn inside the
```
I'd write "drawn mostly inside" instead of "drawn inside".

dbrgn commented on SPECIFICATION.md:
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
+
+Typical glyphs use the X axis (Y=0) as the baseline and are drawn inside the
+area (0,0)..(9,9). Coordinates outside this area are allowed, e.g. for ascenders
+and descenders. And typical glyphs are left aligned to the Y axis (X=0).
```
I'd remove the "And".

dbrgn commented on SPECIFICATION.md:
```diff
@@ -101,17 +101,23 @@ the number may be omitted. If the integer part is zero and there is a fractional
+Typical glyphs use the X axis (Y=0) as the baseline and are drawn inside the
+area (0,0)..(9,9). Coordinates outside this area are allowed, e.g. for ascenders
+and descenders. And typical glyphs are left aligned to the Y axis (X=0).
+Otherwise it influences letter spacing, see section "Letter Spacing" for details.
```
Maybe write "Glyphs that are not aligned to the Y axis have an impact on letter spacing, (...)" instead?

dbrgn commented:
By the way, would be great if the changes were detailed enough to be able to close #5.

ubruhin commented:
Added image and moved coordinate system specification to a separate section to not clutter up the polyline section too much -> [rendered](https://github.com/fontobene/fontobene/blob/40bf0ab74ba3efb1bf25f4b62c2c6fb86f44dd5b/SPECIFICATION.md)
